File: tests/fixtures/baseline/csharp_recovery/Broken.cs
using System;

public class Broken
{
    private static string HelperBrokenCs()
    {
        return "hello";
    }

    public string GreetBrokenCs()
    {
        return HelperBrokenCs();
    // Intentionally missing the closing brace for GreetBrokenCs.

    public string FallbackBrokenCs()
    {
        return HelperBrokenCs();
    }
}

File: tests/fixtures/baseline/java_recovery/Broken.java
import java.util.List;

public class Broken {
    private static String helperBrokenJava() {
        return "hello";
    }

    public String greetBrokenJava() {
        return helperBrokenJava();
    // Intentionally missing the closing brace for greetBrokenJava.

    public String fallbackBrokenJava() {
        return helperBrokenJava();
    }
}
